apps/server/src/modules/tool/external-tool/controller/api-test/tool-configuration.api.spec.ts

```diff
@@ -633,8 +692,58 @@ describe('ToolConfigurationController (API)', () => {
 				const response: Response = await loggedInClient.get(
 					`context-external-tools/${contextExternalTool.id}/configuration-template`
 				);
+
 				expect(response.status).toEqual(HttpStatus.NOT_FOUND);
 			});
 		});
 	});
+
+	describe('GET tools/context-types', () => {
+		describe('when user is not authorized', () => {
+			const setup = async () => {
+				const { teacherAccount, teacherUser } = UserAndAccountTestFactory.buildTeacher();
+
+				await em.persistAndFlush([teacherAccount, teacherUser]);
+				em.clear();
+
+				const loggedInClient: TestApiClient = await testApiClient.login(teacherAccount);
+
+				return { loggedInClient };
+			};
+
+			it('should return unauthorized status', async () => {
+				const { loggedInClient } = await setup();
+
+				const response = await loggedInClient.get('context-types');
+
+				expect(response.status).toEqual(HttpStatus.UNAUTHORIZED);
```

Comment:
unauthorized is returned when you are not authenticated (no credentials) and you don't login when you try to be unauthorized. this test should fail. When a user is not permitted to perform an action then you get a forbidden. https://auth0.com/blog/forbidden-unauthorized-http-status-codes/#:~:text=401%20Unauthorized%20is%20the%20status,an%20action%20on%20a%20resource.